cli/pcluster/configure/networking.py

```diff
@@ -70,29 +71,38 @@ def _create(self, vpc_id, vpc_cidr, subnet_cidrs, internet_gateway_id, compute_s
     def _build_cfn_param(key, value):
         return {"ParameterKey": key, "ParameterValue": value}
 
-    @staticmethod
-    def _get_cfn_parameters(vpc_id, internet_gateway_id):
-        availability_zone = _get_availability_zone()
+    def _get_cfn_parameters(self, vpc_id, internet_gateway_id):
+        availability_zone = self._get_availability_zone()
         return [
             BaseNetworkConfig._build_cfn_param("AvailabilityZone", availability_zone),
             BaseNetworkConfig._build_cfn_param("InternetGatewayId", internet_gateway_id),
             BaseNetworkConfig._build_cfn_param("VpcId", vpc_id),
         ]
 
+    def _get_availability_zone(self):
+        if self.availability_zones is not None:
+            """To Do: is the first az in the azs list the optimal option?"""
+            return self.availability_zones.pop()
+        else:
+            return ""
+
+    # FIXME placeholder for a function that should decide the best availability zone for the current region
```

Comment:
This should be a to-do comment. FIXME should not be present in stable code. Alternatively I would prefer to track this question and the previous optimal AZ comment together as "potential improvements" internally.